client/my-sites/plans/plan-overview/plan-remove/index.jsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Dialog from 'components/dialog';
+import { isInGracePeriod } from 'lib/plans';
+import paths from '../../paths';
+
+const PlanRemove = React.createClass( {
+	propTypes: {
+		plan: React.PropTypes.object.isRequired,
+		selectedSite: React.PropTypes.oneOfType( [
+			React.PropTypes.object,
+			React.PropTypes.bool
+		] ).isRequired
+	},
+
+	getInitialState() {
+		return {
+			showDialog: false
+		};
+	},
+
+	closeDialog( action ) {
+		this.setState( { showDialog: false } );
+
+		if ( action === 'remove' ) {
+			// TODO: Add action that calls the API and cancels the free trial
+
+			// TODO: Display success notice
+
+			page( paths.list( this.props.selectedSite.slug ) );
+		}
+	},
+
+	showDialog( event ) {
+		event.preventDefault();
+
+		this.setState( { showDialog: true } );
+	},
+
+	renderCard() {
+		return (
+			<CompactCard className="plan-remove">
+				<strong>{ this.translate( 'Not looking to purchase?' ) }</strong>
```

Comment:
One benefit of the method that @stephanethomas chose is that the translation strings are less fragile – if we need to change one of the strings, the other doesn't need to be retranslated.

I think the point you raise is a good one (and one to which I definitely don't know the answer): will these sentences, translated separately, make sense in this order in other languages?